csaf_2.1/prose/edit/src/tests-02-optional.md

```diff
@@ -645,3 +645,32 @@ The relevant path for this test is:
 > The key `custom_property` is not defined in the JSON schema.
 
 > A tool MAY remove such keys as a quick fix.
+
+### Same Timestamps in Revision History
+
+It MUST be tested that the timestamps of all items in the revision history are pairwise disjoint, when converted to UTC.
```

Comment:
Could we write instead:

>When converted to UTC, the timestamps of all items in the revision history MUST be pairwise disjoint.

Reply:
Usually, we include the phrase "it MUST be tested". Could we write:

```suggestion
It MUST be tested that the timestamps converted into UTC of all items in the revision history are pairwise disjoint.
``

Reply:
Or:
```suggestion
It MUST be tested that the timestamps, when converted into UTC, of all items in the revision history are pairwise disjoint.
```

Reply:
Or:
```suggestion
It MUST be tested that, when converted into UTC, the timestamps of all items in the revision history are pairwise disjoint.
```

Reply:
I like to keep the focus on the compliance fact it self and not the on out of some means of compliance: My understanding is here that we provide the test actor a list of claims they can validate and not so much a MUST on theses actors to test. The motivation to test I see as external to our specifications. But, I may be wrong.

Reply:
@sthagen: The reason, why I would like to add the "It MUST be tested that" is that we use it in almost all tests in CSAF 2.0 (cf. https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#618-invalid-cvss and following). I think removing the phrase might raise questions if the implementers have to do different things here (which is not the case).

Reply:
We could put that on the table for our native speakers today.

Reply:
Thank you for adding to discussion. I think we should remove that MUST on the test activity to the MUST on the properties of the document (or where applicable to the behaviors of the roles under validation) instead

Reply:
@sthagen: As we discussed in today's call (TC meeting) regarding the phrase "it MUST be tested that" that it affects all test. Therefore, IMHO it should be handled in a separate issue. This would allow us to process this PR. Do you agree?

Reply:
The remaining loophole for inconsistencies sneaking into implementations now is IMO the "when converted to", what if no conversion is done and consumers use these values as mere textual identifiers which can be have as many colliding values as timezones are available?
But, in case this is also spread across other sections we can easily also split dealing with that concern into another task (PR).

Reply:
Thanks. This is addressed in [f0d17e9](https://github.com/oasis-tcs/csaf/pull/744/commits/f0d17e9dda3b2a92eec60793752c9861dd0dc5e7)